tests/test_client_functional.py

```diff
@@ -82,6 +83,77 @@ async def on_reuseconn(session, ctx, params):
     await resp2.read()
 
     assert 1 == cnt_conn_reuse
+    await client.close()
```

Comment:
This shouldn't be needed, our fixtures handle closing the client.

Reply:
Will validate and remove

Reply:
removed in a7dc41ce44c4c735c6a98234ab2c072c95e7d33a